task_12/html-dev/index.html

```diff
@@ -0,0 +1,136 @@
+  <link href="https://fonts.googleapis.com/css?family=Lobster|Fira+Mono" rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/style.css">
+  <script src="assets/js/script.js" defer></script>
+</head>
+
+<body>
+  <header>
+    <div class="clock">
+      <span class="icon" aria-hidden="true">🕙</span>
+      <time id="clock"></time>
+    </div>
+    <a href="/" class="logo" aria-label="home">
+      <img src="assets/img/logo.png" alt="logo" class="logo--img">
+    </a>
+    <div class="login-signup">
+      <button type="button" class="login-signup--btn">
+        <span>
+          <span class="icon" aria-hidden="true">👨🏻‍🍳</span>
+          Login / Signup
+        </span>
+      </button>
+    </div>
+  </header>
+  <main>
+    <button class="btn__add-pizza">
+      <span>
+        <span class="icon" aria-hidden="true">🍕</span>
+        Add new pizza
+      </span>
+    </button>
+    <section class="pizzas">
+      <div class="pizza">
+        <img src="assets/img/pizza1.jpg" alt="pizza1" class="pizza--picture">
+        <time class="pizza--timestamp">10:52:03</time>
+        <span class="pizza--queue-number">#1</span>
+        <span class="pizza--eta">ETA:
+          <time datetime="04:00">1 min</time>
+        </span>
+        <span class="pizza--price">$3.40</span>
+      </div>
+      <div class="pizza">
+        <img src="assets/img/pizza2.jpg" alt="pizza2" class="pizza--picture">
+        <time class="pizza--timestamp">10:53:23</time>
+        <span class="pizza--queue-number">#2</span>
+        <span class="pizza--eta">ETA:
+          <time datetime="04:00">2 min</time>
+        </span>
+        <span class="pizza--price">$3.50</span>
+      </div>
+      <div class="pizza">
+        <img src="assets/img/pizza3.jpg" alt="pizza3" class="pizza--picture">
+        <time class="pizza--timestamp">10:54:02</time>
+        <span class="pizza--queue-number">#3</span>
+        <span class="pizza--eta">ETA:
+          <time datetime="04:00">4 min</time>
+        </span>
+        <span class="pizza--price">$3.30</span>
+      </div>
+      <div class="pizza">
+        <img src="assets/img/pizza4.jpg" alt="pizza4" class="pizza--picture">
+        <time class="pizza--timestamp">10:55:54</time>
+        <span class="pizza--queue-number">#4</span>
+        <span class="pizza--eta">ETA:
+          <time datetime="04:00">5 min</time>
+        </span>
+        <span class="pizza--price">$4.20</span>
+      </div>
+      <div class="pizza">
+        <img src="assets/img/pizza5.jpg" alt="pizza5" class="pizza--picture">
+        <time class="pizza--timestamp">10:56:13</time>
+        <span class="pizza--queue-number">#5</span>
+        <span class="pizza--eta">ETA:
+          <time datetime="04:00">11 min</time>
+        </span>
+        <span class="pizza--price">$3.50</span>
+      </div>
+      <div class="pizza">
+        <img src="assets/img/pizza6.jpg" alt="pizza6" class="pizza--picture">
+        <time class="pizza--timestamp">10:56:22</time>
```

Comment:
It's better to use `datetime` attribute with time tag.

Time element developed to determine actual time when displaying relative one, i. e. we may have displayed text _yesterday_, but it was actually _2018-01-27 16:03:18_.

---

Learn BEM more deeply. _M_ stands for modifier. Modifiers represent state changes of a block or an element. Examples:
- `card` — a container, i.e. a block
- `card__content` child element of `card`, i.e. an element or an inner block
- `card--opened` — modifier for `card`
- `card__content--empty` — empty `card__content` element, modifier

Reply:
Well, that's embarrassing — I totally mixed up the order of dashes and underscores :) Thought it was `block--element__modifier`. 

Thanks for pointing it out, and thanks for the review. I think I'll fix the code tomorrow and add a commit to this PR.